app/ai/voice/agents/breeze_buddy/agent/flow.py

```
@@ -24,14 +24,119 @@
+    if lead.metaData and lead.metaData.get("playground"):
+        logger.info(f"Loading playground template for lead {lead.id}")
+
+        # Build template model from playground metadata
+        flow = lead.metaData.get("flow", {})
+        configurations_dict = lead.metaData.get("configurations")
+
+        # Parse configurations dict into ConfigurationModel if present
+        if configurations_dict and isinstance(configurations_dict, dict):
+            try:
+                configurations = ConfigurationModel(**configurations_dict)
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)